### PR TITLE
Openstack wrapper

### PR DESCRIPTION
**Pull request by sanjaymadane**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/openstack-wrapper/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,274 @@
+// Type definitions for openstack-wrapper 2.1.6
+// Project: https://www.npmjs.com/package/openstack-wrapper
+// Definitions by: Sanjay Madane <https://github.com/sanjaymadane>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+declare module "openstack-wrapper" {
```
Comment (mhegazy) [inline types/openstack-wrapper/index.d.ts]: do not wrap in a module declaration, i.e. `declare module "openstack-wrapper" { ...`. instead, export declarations directelly:

```ts
 export interface RequestOption {
...
```

File: types/openstack-wrapper/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+        neutron: Neutron;
+        nova: Nova;
+        octavia: Octavia;
+    }
+
+    export class getSimpleProject{
+        constructor(username: string, password: string, project_id: string, keystone_url: string, cb: Function);
+    }
+}
+export default "openstack-wrapper"
```
Comment (mhegazy) [inline types/openstack-wrapper/index.d.ts]: is the default export of this module a string?

**Review by mhegazy — DISMISSED** [review]

**Review by sanjaymadane — COMMENTED** [review]
Hi, @mhegazy - as per your review comments. updated code and committed.

**Comment by DanielRosenwasser** [comment]
Can you add a `tslint.json` as requested by the checklist? See https://github.com/DefinitelyTyped/DefinitelyTyped/tree/d542dc45448a3759649e27d6579dd5af33928131#lint for more details.

Note that the `Function` type is not allowed by the current lint rules. If you could provide a more accurate function type, or use `(...args: any[]) => any` instead, that would be better. If not, you can disable that lint rule.

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]
Can you add a `tslint.json` as requested by the checklist? See https://github.com/DefinitelyTyped/DefinitelyTyped/tree/d542dc45448a3759649e27d6579dd5af33928131#lint for more details.

Note that the `Function` type is not allowed by the current lint rules. If you could provide a more accurate function type, or use `(...args: any[]) => any` instead, that would be better. If not, you can disable that lint rule.

File: types/openstack-wrapper/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (mhegazy) [inline types/openstack-wrapper/tsconfig.json]: can you also make this `true` please.

**Review by DanielRosenwasser — APPROVED** [review]

**Pull request merged**